app/javascript/member-facing/backbone/petition.js

```diff
@@ -17,6 +17,13 @@ const Petition = Backbone.View.extend({
     this.followUpUrl = options.followUpUrl;
     this.submissionCallback = options.submissionCallback;
     this.skipOnSuccessAction = options.skipOnSuccessAction;
+
+    if (options.redirectAfterAction !== undefined) {
+      this.redirectAfterAction = options.redirectAfterAction;
+    } else {
+      this.redirectAfterAction = true;
+    }
+
```

Comment:
Can you explain why is this necessary? What am I missing (keep in mind I've just had a pint)?